### PR TITLE
[t2434] WIP - using transform: translate

### PR DESCRIPTION
**Pull request by secretrobotron**

### CODE REVIEW

File: src/util/dragndrop.js [hunk 1]
```diff
@@ -35,17 +39,39 @@ define( [ 'core/eventmanager' ], function( EventManager ) {
...
+  function __getScrollGroup( scrollElement ) {
+    var i, newScrollGroup;
+
+    if ( scrollElement ) {
+      for ( i = __scrollGroups.length - 1; i >= 0; --i ) {
+        if ( __scrollGroups[ i ].scrollElement === scrollElement ) {
+          return __scrollGroups[ i ];
+        }
+      }
+      newScrollGroup = new ScrollGroup.ScrollGroup( scrollElement );
```
Comment (ScottDowne) [inline src/util/dragndrop.js:53]: new ScrollGroup.ScrollGroup?

That doesn't look right.

Comment (secretrobotron) [inline src/util/dragndrop.js:53]: No? What should it be?

File: templates/basic/saved-data.json [hunk 2]
```diff
@@ -70,33 +70,6 @@
...
-                  "italics": false,
-                  "underline": false
-                },
-                "left": 25,
-                "top": 0,
-                "width": 50,
-                "zindex": 1000
-              },
-              "track": "Layer",
-              "name": "TrackEvent4"
```
Comment (ScottDowne) [inline templates/basic/saved-data.json]: Is there a reason you deleted this?

Comment (jbuck) [inline templates/basic/saved-data.json]: Yeah, I don't see the reason for deleting this. Please re-add

File: src/util/lang.js [hunk 3]
```diff
@@ -156,6 +156,23 @@ define( [ "./shims" ], function(){
       element.removeEventListener( "transitionend", listener, false );
       element.removeEventListener( "oTransitionEnd", listener, false );
       element.removeEventListener( "webkitTransitionEnd", listener, false );
+    },
+
+    setTransformProperty: function( element, transform ) {
```
Comment (jbuck) [inline src/util/lang.js:174]: I wonder if it wouldn't be better to figure out what prefix to use ahead of time. MDN suggestions a solution if you're interested: https://developer.mozilla.org/en-US/docs/CSS/CSS_animations/Detecting_CSS_animation_support

**Pull request merged**